src/Ghcid.hs

```diff
@@ -375,21 +378,35 @@ runGhcid session waiter termSize termOutput opts@Options{..} = do
             currTime <- getShortTime
             if not $ null restartChanged then do
                 -- exit cleanly, since the whole thing is wrapped in a forever
-                unless no_status $ outputFill currTime Nothing $ "Restarting..." : map ("  " ++) restartChanged
+                unless no_status $ outputFill currTime Nothing evals $ "Restarting..." : map ("  " ++) restartChanged
                 return Continue
             else do
-                unless no_status $ outputFill currTime Nothing $ "Reloading..." : map ("  " ++) reason
+                unless no_status $ outputFill currTime Nothing evals $ "Reloading..." : map ("  " ++) reason
                 nextWait <- waitFiles waiter
                 fire nextWait =<< sessionReload session
 
     fire nextWait (messages, loaded)
 
 
 -- | Given an available height, and a set of messages to display, show them as best you can.
-prettyOutput :: String -> Int -> [Load] -> [String]
-prettyOutput currTime loadedCount [] =
-    [allGoodMessage ++ " (" ++ show loadedCount ++ " module" ++ ['s' | loadedCount /= 1] ++ ", at " ++ currTime ++ ")"]
-prettyOutput _ _ xs = concatMap loadMessage xs
+prettyOutput :: String -> Int -> [Load] -> [((FilePath, (Int, Int)), (String, String))] -> [String]
+prettyOutput currTime loadedCount [] evals =
+    [allGoodMessage ++ " (" ++ show loadedCount ++ " module" ++ ['s' | loadedCount /= 1] ++ ", at " ++ currTime ++ ")"
+    ] ++ concatMap printEval evals
+prettyOutput _ _ xs evals = concatMap loadMessage xs ++ concatMap printEval evals
+
+printEval :: ((FilePath, (Int, Int)), (String, String)) -> [String]
+printEval ((file, (line, col)), (msg, result)) = do
+  [ " "
+    , mconcat
+        [ file
+        , ":"
+        , show line
+        , ":"
+        , show col
+        ]
+    ] ++ (zipWith (++) (repeat "λ> ") $ lines msg)
```

Comment:
Normally I try and avoid Unicode since otherwise you are causing issues for a set of people who would otherwise have had their code work. Why not make this `$>` instead?